app/controllers/configuration_controller.rb

```diff
@@ -115,6 +115,7 @@ def view_selected
     @changed = session[:changed]
     render :update do |page|
       page << javascript_prologue
+      page << javascript_show("buttons_off")
```

Comment:
This change would simply just call `$('#buttons_off').show();` before the whole `page.replace(...)`.
This doesn't really fix the problem as described in the bug report: the Save & Reset buttons should
be activated / deactived depending on whether or not any changes in the form have been made.